lib/graph-document.js

```diff
@@ -21,92 +22,100 @@ function node (state, createEdge) {
   var entry = utils.basefile(state.metadata.entry)
   var self = this
 
-  serverRender(entry, function (err, render) {
-    if (err && err.isSsr) {
-      self.emit('ssr', {success: false, error: err})
-    } else if (err) {
-      err = ttyError('documents', entry, err)
-      return self.emit('error', 'documents', entry, err)
-    }
-
-    self.emit('ssr', {success: true, renderRoute: renderRoute})
+  var ssr = new ServerRender(entry)
+  if (ssr.error && ssr.error.isSsr) {
+    self.emit('ssr', { success: false, error: ssr.error })
+  } else if (ssr.error) {
+    ssr.error = ttyError('documents', entry, ssr.error)
+    return self.emit('error', 'documents', entry, ssr.error)
+  }
```

Comment:
are these error checks still doing anything at this point? since they are run sync after instantiating the ServerRender class

Reply:
uhhhhhh, I don't know - probably not hey

Reply:
lets get rid of it & then merge if you think it's ready?

Reply:
oh, the `.error` property actually is sync there. nvm!